http-api/src/v1/projects.rs

```diff
@@ -380,7 +381,7 @@ async fn blob_handler(
     let path = path.strip_prefix('/').ok_or(Error::NotFound)?.to_string();
     let Query(query) = query.unwrap_or_default();
     let theme = if query.highlight {
-        Some(ctx.theme.as_str())
+        Some(query.theme.as_deref().unwrap_or("base16-ocean.dark"))
```

Comment:
I like this idea, but I would leave the theme in the context as the default/fallback. That way it can still be configured when nothing's passed in the request.

Reply:
What would be the use case? There has to be some kind of front-end through which we'd look at the code anyway and then the frontend should always be able to control what theme to show, no?

Reply:
Well the frontend doesn't necessarily know the available themes right? There has to be a way to just ask for the default theme, which currently you do by not setting `theme=`, but then the question is who sets the default theme?

Reply:
By default the API responds without including any colors. You have to explicitly set `?highlight=true`, and in that case I assume the frontend knows about color themes and would pass in the theme that it requires. You're right in that we don't have an endpoint listing available themes, but since `radicle-interface` is the only consumer at the moment, we can just assume the frontend knows what themes are available. Wouldn't falling back to the dark theme as we do now be enough?

If you really insist, I can add back the CLI param. I just don't fully understand why a node operator would ever want to set the theme color. 🤷

Reply:
> If you really insist, I can add back the CLI param. I just don't fully understand why a node operator would ever want to set the theme color, when the frontend will override it anyway. shrug

Because you made it optional, so the frontend may not override it.. Either make it required, and then we don't need a default, or make it optional, and then we need a way to set the default. Also, if it's non-optional, you may as well collapse the two into a single var, eg. `highlight=base16-ocean.dark`

Reply:
> Also, if it's non-optional, you may as well collapse the two into a single var, eg. highlight=base16-ocean.dark

Yeah, I thought about that, but I didn't want to break the existing UI if we merge and deploy this before the theme PR is merged on interface.

I still think that we don't need to be able to set the default theme via a CLI option, but I'm not going to die on this hill :P

https://github.com/radicle-dev/radicle-client-services/pull/183/commits/8165c6c95a0ad2761d5fc3df0880167f5e749b95